cypress/e2e/internal/monitoring-stations/abstraction-alerts/cancel-alert.cy.js

```diff
@@ -0,0 +1,99 @@
+'use strict'
+
+describe('Monitoring stations - Abstraction alerts (internal)', () => {
```

Comment:
This is what we'll see in the Cypress UI and the report. So, the descriptions need to be distinct and should reflect the scenarion being covered.

```suggestion
describe('Set up but then cancel an abstraction alert (internal)', () => {
```